scripts/utils/codemod/create-processor.js

```diff
@@ -5,7 +5,7 @@ const frontmatter = require('remark-frontmatter');
 const remarkMdx = require('remark-mdx');
 const remarkMdxjs = require('remark-mdxjs');
 
-const createProcessor = ({ codemods }) => {
+const createProcessor = ({ codemods = [] } = {}) => {
```

Comment:
this goes along with my other question!

Reply:
Since I want the stringify module to _not_ use codemods, it felt awkward to call it like: `createProcessor({ codemods: [] })`, so I updated this to make the argument a default value with no plugins. This allows me to call it like `createProcessor()`